### PR TITLE
Make a navigation drawer divider by group

### LINKED ISSUE #124: Make a navigation drawer divider by group
**Idea Description**
Make DrawerGroup and add it in DrawerItem for inserting divider.
like this, 
```kotlin
DrawerItem.Sessions(DrawerGroup.Session, R.string.title_sessions, Icons.Default.Event, SessionsNavGraph.sessionRoute),
```

Currently, this feature is implemented by a if statement in NavigationDrawerItem 
```kotlin
if (drawerItem == DrawerItem.Sessions || drawerItem == DrawerItem.Map) {
```

**Reference images and links**
- [Figma](https://www.figma.com/file/NcSMs6dMsD88d4wOY0g3rK/Conference-App)

screenshot

### PR DESCRIPTION
**Pull request by Kaito-Dogi**
## Issue
- close #124 

## Overview (Required)
- DrawerGroup defined.
- added DrawerGroup to DrawerItem property
- added displaying "divider" when it is not the last of DrawerItem and the last of group.

## Links
- 

## Screenshot
No differences.
Before | After
:--: | :--:
 |

### CODE REVIEW

File: app-android/src/main/java/io/github/droidkaigi/confsched2022/KaigiApp.kt [hunk 1]
```diff
@@ -176,24 +176,55 @@ class KaigiAppScaffoldState @OptIn(ExperimentalMaterial3Api::class) constructor(
...
     companion object {
         fun ofOrNull(route: String): DrawerItem? {
             return values().firstOrNull { it.navRoute == route }
         }
     }
+
+    val isLastItem: Boolean
+        get() = ordinal == DrawerItem.values().lastIndex
+
+    val isGroupLastItem: Boolean
```
Comment (takahirom) [inline app-android/src/main/java/io/github/droidkaigi/confsched2022/KaigiApp.kt:226]: 🆒

**Comment by takahirom** [comment]
Please fix conflicts 🙏

**Comment by Kaito-Dogi** [comment]
Thanks for the review.
Conflicts have been resolved.
Please check back!

**Comment by takahirom** [comment]
Thanks!

**Pull request merged**